File: main/scala/mod/iceandshadow3/basics/BItemProperty.java
package mod.iceandshadow3.basics;

import mod.iceandshadow3.compat.item.CRefItem;
import mod.iceandshadow3.compat.CRefWorld;

public abstract class BItemProperty {
	public abstract String name();
	public abstract float call(CRefItem item, CRefWorld world);
}

File: main/scala/mod/iceandshadow3/basics/BItemProperty.scala
package mod.iceandshadow3.basics

import mod.iceandshadow3.compat.item.CRefItem
import mod.iceandshadow3.compat.CRefWorld

abstract class BItemProperty {
	def name: String
	def call(item: CRefItem, world: CRefWorld): Float
}